src/OpenTelemetry.Instrumentation.MassTransit/Implementation/OperationName.cs

```diff
@@ -0,0 +1,56 @@
+// <copyright file="OperationName.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Instrumentation.MassTransit.Implementation
+{
+    /// <summary>
+    /// MassTransit diagnostic source operation name constants.
+    /// </summary>
+    public static class OperationName
```

Comment:
public intentional?

Reply:
Yep, intentional. It was left public so user can filter logged activities using these constants